packages/vuetify/src/components/VDatePicker/VDatePicker.ts

```diff
@@ -90,6 +90,8 @@ export default mixins(
       type: String,
       default: '$vuetify.datePicker.itemsSelected',
     },
+    // Allow select date with range, must be used together with multiple
+    selectRange: Boolean,
```

Comment:
We can enable multiple automatically if range

Reply:
Anyway, comment isn't very useful here
```suggestion
```

Reply:
Changed to Range and use computed internalMultiple